src/components/AddTaskButton.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { AddTaskBox, TaskTitle } from "./../styled-components/reusables";
+import { Link } from "react-router-dom";
+
+export default function AddTaskButton() {
+	return (
+		<Link className="route-link" to="/add-tasks">
+			<AddTaskBox>
+				<TaskTitle lSpacing="5px">ADD NEW TASK</TaskTitle>
```

Comment:
What do you think about passing the title as a prop vs as a child?

Reply:
I agree I can pass the title in as prop.
Done 👍